assets/js/source_log_chart.jsx

```diff
@@ -76,7 +76,7 @@ const renderElixirLoggerTooltip = ({ data, color }) => {
     { c: brandGray, p: "other", t: "Other" },
   ]
   return (
-    <div style={{ backgroundColor: brandLightBlack }}>
+    <div style={{ backgroundColor: brandLightBlack, padding: "6px" }}>
```

Comment:
This is fine, but can be replaced with a `tw-p-2` class

Reply:
would this work? since this is actually changing the style instead of the class I avoided tailwind